units/host_status.go

```diff
@@ -180,3 +189,38 @@ func setCloudHostStatus(ctx context.Context, m cloud.Manager, h host.Host, hostS
 	})
 	return nil
 }
+
+func initialSetup(ctx context.Context, cloudMgr cloud.Manager, h *host.Host) error {
```

Comment:
Can we make these methods on the job? It seems like of weird to have a function called `initialSetup` that applies to the entire `units` package.

Reply:
done